HelloPush/app/build.gradle

```diff
@@ -34,11 +31,11 @@ android {
 }
 
 dependencies {
-    compile 'com.android.support:appcompat-v7:25.0.0'
-    compile 'org.jboss.aerogear:aerogear-android-core:3.0.0'
-    compile 'org.jboss.aerogear:aerogear-android-pipe:3.0.0'
-    compile 'org.jboss.aerogear:aerogear-android-push:4.0.1'
-    compile 'com.google.firebase:firebase-messaging:9.8.0'
+    compile 'com.android.support:appcompat-v7:26.0.1'
+    compile 'org.jboss.aerogear:aerogear-android-core:3.1.0'
+    compile 'org.jboss.aerogear:aerogear-android-pipe:3.2.0'
+    compile 'org.jboss.aerogear:aerogear-android-push:4.2.0'
+    compile 'com.google.firebase:firebase-messaging:11.0.4'
```

Comment:
Yes, but it's a long history, it was a but when we were using maven to release our libraries and the aar did not download the dependency itself.